### DISCUSSION TITLE
Error when using Endnote datafile

### DISCUSSION THREAD

**Discussion by Marielle66** [asreview/asreview#1693] [Q&A]
I'm trying to use an exportfile from Endnote, but keep getting the same error message:
Failed to import file 'Orthostatische_hypotensie.txt'. Error reading RIS file: Expected start tag in line 0: (1968). "A nonnarcotic analgesic agent: methotrimeprazine (levoprome)." Jama 204(2): 161-162. Please try again. 

Any suggestions on what I might do wrong?

**Comment by J535D165** [comment]
Can you open the text file in a text editor and show me the first 10-20 lines?

If that doesn't work, can you share the file here or via email (asreview@uu.nl)?

**Reply by Marielle66** [reply]
This is what the exportfile looks like:
[Orthostatische hypotensie.txt](https://github.com/asreview/asreview/files/14405467/Orthostatische.hypotensie.txt)

(1968). "A nonnarcotic analgesic agent: methotrimeprazine (levoprome)." Jama 204(2): 161-162.
	
(1969). "A second report on Levodopa." Med Lett Drugs Ther 11(18): 73-75.
	
(1970). "L-dopa. II." Br Med J 4(5734): 541-542.
	
(1970). "[Orthostatic hypotension caused by diuretics]." Tidsskr Nor Laegeforen 90(9): 881-882.
	
(1970). "Specific therapy in Parkinson's disease." Med J Aust 2(14): 612-613.
	
(1971). "Evaluation of Mesoridazine. (Serentil)." Jama 216(2): 313-314.
	
(1975). "Individualization of drug therapy for the parkinsonian patient." Jama 233(11): 1198-1201.
	
(1977). "Labetalol in hypertension." Lancet 1(8022): 1147-1148.
	
(1977). "Prazosin (Minipress) for hypertension." Med Lett Drugs Ther 19(1): 1-2.
	
(1978). "Drugs causing postural hypotension." Med Lett Drugs Ther 20(3): 15-16.
	
(1978). "Labetalol for hypertension." Drug Ther Bull 16(23): 89-90.
	
(1988). "Medical treatment of benign prostatic hyperplasia." Lancet 1(8594): 1083-1084.
	
(1989). "Alpha-blockade for hypertension." Lancet 1(8652): 1448-1449.


(These first lines do not have an abstract because they are to old)

**Comment by Rensvandeschoot** [comment]
Can you export the file in the RIS file format?

**Reply by Marielle66** [reply]
![image](https://github.com/asreview/asreview/assets/161308956/e0207c46-966b-46cc-982d-13c04923511b)
These are the only options I get in Endnote. Might the fact that I use a free trial version of Endnote be the cause?

**Reply by FelixWdm** [reply]
@Marielle66 check the settings here: https://libguides.vu.nl/c.php?g=616082&p=4691268. The output style should be RIS.

**Reply by FelixWdm** [reply]
![afbeelding](https://github.com/asreview/asreview/assets/49488769/3485bce3-edff-4642-bbde-fa97f75ec1c7)

**Comment by Marielle66** [comment]
Aaah, thanks! It wasn't in my default list of export styles. I have added it now and I am going to try again. Hope it works now.

**Reply by FelixWdm** [reply]
No EndNote doesn't make it easy to export in the most used format...

**Comment by Marielle66** [comment]
Still no luck :-(

Failed to import file 'Orthostatische_hypotensie.txt'. Error reading RIS file: Expected start tag in line 0: (1968). "A nonnarcotic analgesic agent: methotrimeprazine (levoprome)." Jama 204(2): 161-162. Please try again.

What I did:
![image](https://github.com/asreview/asreview/assets/161308956/000b5573-e468-4f56-8ef0-000a8fab5f1d)
And then:
![image](https://github.com/asreview/asreview/assets/161308956/94bf5c39-8a10-414b-8b14-45631ff5c407)

**Comment by Marielle66** [comment]
Somehow Endnote does not add  the labels to my export file. Tried everything but cannot find the solution.

I found another solution which works but is not very user-friendly. I set the export style on 'Tab Delimited' and used 'Copy Formatted references' in the reference menu. I pasted the library in excel and added the labels 'title' and 'abstract' above the right columns. I saved the dataset as csv-file and added that file to ASreview lab. Then it works perfectly.

Thanks everybody for your help! If you find another solution I would love to hear it!

**Reply by FelixWdm** [reply]
Now I'm wondering what the import file looks like, was that CSV or RIS as well? Can you share the TXT file you exported?

**Reply by Marielle66** [reply]
The file I imported in ASReview Lab was a CSV-file. I have added both files. de txt-file is what I exported from Endnote and imported in Ex
[Orthostatische hypotensie.txt](https://github.com/asreview/asreview/files/14420166/Orthostatische.hypotensie.txt)
cel. De csv is the file I imported in ASReview Lab
[OH.csv](https://github.com/asreview/asreview/files/14420163/OH.csv)

**Reply by FelixWdm** [reply]
Thanks! I'm guessing now, the fields to create a full RIS file are not available in EndNote, did you also import a CSV file into EndNote in the first place?

**Reply by Marielle66** [reply]
No, I used data from Pubmed and the only file that I could import into Endnote that also had an abstract was a text-file (the 'save' option 'Pubmed' in Pubmed). Pubmed allows you to export a csv-file but I could not import this in Endnote.

Endnote shows the fieldnames in the application itself but doesn't include them in the exportfile.

**Comment by FelixWdm** [comment]
Last check, in PubMed, you should select 'Send to' and then 'Citation manager' 

![afbeelding](https://github.com/asreview/asreview/assets/49488769/f921e91c-4bad-41f6-9c73-9f0465c05656)

EndNote has many custom settings, most likely something went wrong there.

**Reply by FelixWdm** [reply]
I just checked the PubMed -> EndNote -> RIS workflow and I do get abstracts with default settings

**Reply by Marielle66** [reply]
Thanks!! That did the trick. I have been trying everything, everywhere, all at once so the most obvious pathway got lost in translation. This works perfect now.
